src/actions/actionCreators.js

```diff
@@ -0,0 +1,11 @@
+const increment = index => ({
+  type: 'INCREMENT_LIKES',
```

Comment:
@gbols You can create an `actionType` file for the type so we all can follow that convention instead of hard-coding the type.